conbench/entities/benchmark_result.py

```diff
@@ -72,26 +84,59 @@ def create(data):
 
         # defaults
         benchmark_result_data = {}
-        complete_iterations = False
+
+        # Note(JP): if no data was provided, should this stay `True`?
+        # We will see. https://github.com/conbench/conbench/issues/803.
+        any_errored_iteration = True
```

Comment:
I renamed `complete_iterations` to `any_errored_iteration` (boolean inverse). As an outsider, the new variable name is much clearer to me.